File: av1/mdc-java/MDC/MDC.java
public class MDC {
    public static int mdc(int a, int b) {
        while (b != 0) {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }

    public static void main(String[] args) {
        int numero1 = 36;
        int numero2 = 48;
        int resultadoMDC = mdc(numero1, numero2);
        System.out.println("O MDC de " + numero1 + " e " + numero2 + " é " + resultadoMDC);
    }
}

File: av1/mdc-js/mdc.js
function mdc(a, b) {
    while (b !== 0) {
        const temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

const numero1 = 36;
const numero2 = 48;
const resultadoMDC = mdc(numero1, numero2);
console.log(`O MDC de ${numero1} e ${numero2} é ${resultadoMDC}`);